docs/UserGuide.md

```diff
@@ -245,20 +245,24 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL_ADDRESS r/ROLE et/EMPLOYMENT_TYPE s/E
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 Providing multiple tag values in a single `t/` prefix will lead to an error. (i.e. `add n/John p/90909090 e/john@gmail.com r/Software Tester et/Full time s/4500 l/High School y/3 t/smart helpful` will lead to an error)
 </div>
-* Inputs for each prefix is taken as a single value. (i.e. `r/software engineer` has the value `software engineer`)
+* Parameters for each prefix is taken as a single value. (i.e. `r/software engineer` has the value `software engineer`)
 
 Examples:
 * `add n/Bob p/87654321 e/bob@gmail.com r/Software Engineering et/Full time s/4000 l/High School y/2 i/2021-10-21, 20:00 nt/This applicant has the credentials needed for this job.`
 * `add n/John p/90909090 e/john@gmail.com r/Software Tester et/Full time s/4500 l/High School y/3 t/smart t/helpful` to add a person named `John` with two tags `smart` and `helpful`
 
+Before: ![images](images/AddBefore.png)
```

Comment:
If it is possible, could you change all the mockup's font to be Montserrat, with either size 11 or 12, as I followed the sizing and font from Chen En's mockups so that it looks like the same person did all of the mockups.